### PR TITLE
Add release-it to simplify the release process

### PR DESCRIPTION
**Pull request by sdebarros**
Using template from https://github.com/rwjblue/create-rwjblue-release-it-setup. This automates the process of updating the CHANGELOG, creating the tag and the Github release, the npm publishing is still done by the CI.

### CODE REVIEW

File: .release-it.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "release-it-lerna-changelog": {
+      "infile": "CHANGELOG.md"
+    },
+    "release-it-yarn-workspaces": {
+      "publish": false
+    }
+  },
+  "git": {
+    "commitMessage": "v${version}"
+  },
```
Comment (Turbo87) [inline .release-it.json:12]: I would recommend https://github.com/simplabs/qunit-dom/blob/master/.release-it.js#L11 to match what we're currently doing here

File: package.json [hunk 2]
```diff
@@ -4,13 +4,16 @@
...
   "workspaces": [
     "packages/*"
   ],
   "devDependencies": {
     "@mermaid-js/mermaid-cli": "^8.4.8",
-    "lerna-changelog": "^1.0.1"
+    "lerna-changelog": "^1.0.1",
+    "release-it": "^14.2.0",
+    "release-it-yarn-workspaces": "^2.0.0"
   }
```
Comment (Turbo87) [inline package.json:18]: release-it-lerna-changelog missing here?

File: .release-it.json (hunk 1 above)
Comment (sdebarros) [reply .release-it.json:12]: Our tags so far have been without the `v` 😬 So I was maintaining that, but I'm fine with changing it.

Comment (Turbo87) [reply .release-it.json:12]: 😱 sorry, nevermind then 😅

File: package.json (hunk 2 above)
Comment (sdebarros) [reply package.json:19]: Added.

**Review by Turbo87 — APPROVED** [review]

**Pull request merged**